cli/integrity/integrity_test.go

```diff
@@ -0,0 +1,62 @@
+package integrity_test
+
+import (
+	"testing"
+
+	"github.com/spf13/cobra"
+	"github.com/stretchr/testify/require"
+	"github.com/tarantool/tt/cli/integrity"
+)
+
+func TestNewSigner(t *testing.T) {
+	testCases := []struct {
+		name           string
+		privateKeyPath string
+	}{
+		{
+			name:           "Empty path",
+			privateKeyPath: "",
+		},
+		{
+			name:           "Arbitrary path",
+			privateKeyPath: "private.pem",
+		},
+	}
+
+	for _, testCase := range testCases {
+		t.Run(testCase.name, func(t *testing.T) {
+			signer, err := integrity.NewSigner(testCase.privateKeyPath)
+			require.Nil(t, signer, "signer must not be created")
+			require.Error(t, err, "an error should be produced")
```

Comment:
Let's compare the error message too.

Reply:
Fixed

Reply:
nit: Errorf asserts that a function returned an error (i.e. not `nil`).

Reply:
`Errorf`->`EqualError`.

Reply:
Right. Fixed